src/components/bcros/filing/common/HeaderActions.vue

```diff
@@ -52,21 +52,42 @@
         </div>
       </template>
     </BcrosDialog>
-    <!-- the main button -->
+
+    <!-- the main action button -->
     <UButton
       variant="ghost"
       class="px-3 py-2"
       data-cy="filing-main-action-button"
       @click="handleButtonClick"
     >
+      <!-- filings available on paper only show Request a Copy / Close button -->
       <template v-if="filing.availableOnPaperOnly">
         <strong v-if="!isExpanded">{{ $t('button.filing.actions.requestACopy') }}</strong>
         <strong v-else>{{ $t('button.filing.actions.close') }}</strong>
       </template>
+
+      <!-- staff filings, except Court Orders, show View / Hide button -->
       <template v-else-if="isTypeStaff && !isFilingType(filing, FilingTypes.COURT_ORDER)">
         <strong v-if="!isExpanded">{{ $t('button.filing.actions.view') }}</strong>
         <strong v-else>{{ $t('button.filing.actions.hide') }}</strong>
       </template>
+
```

Comment:
Below, I thought it was clearer to have separate conditions, even though they all display View/Hide, instead of trying to combine the conditions. Thoughts?